### PR TITLE
Show duplicate lines

### PR DESCRIPTION
**Pull request by pit00**
Keep the unique lines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Sort lines of text in Visual Studio Code. The following types of sorting are sup
 | sortLines.sortLinesUnique | Sort lines (unique ascending, case sensitive) | Regular character code keeping only unique items
 | sortLines.sortLinesShuffle | Sort lines (shuffle) |
 | sortLines.removeDuplicateLines | Sort lines (remove duplicate lines)
+| sortLines.showDuplicateLines | Sort lines (remove duplicate lines)
```
Comment (mrienstra) [inline README.md:19]:
```suggestion
| sortLines.showDuplicateLines | Sort lines (show duplicate lines)
```

File: package.json [hunk 2]
```diff
@@ -92,6 +93,10 @@
         "command": "sortLines.removeDuplicateLines",
         "title": "Sort lines (remove duplicate lines)"
       },
+      {
+				"command": "sortLines.showDuplicateLines",
+				"title": "Sort lines (show duplicated lines)"
```
Comment (mrienstra) [inline package.json:98]:
```suggestion
				"title": "Sort lines (show duplicate lines)"
```

**Review by mrienstra — COMMENTED** [review]
Nice, I had a use case for this recently.

Maybe make it more clear that it removes unique lines (assuming I got that right), so instead of "Sort lines (show duplicated lines)", maybe something like "Sort lines (remove unique lines)" or "Sort lines (only keep duplicate lines)" or "Sort lines (remove non-duplicate lines)".

**Comment by pit00** [comment]
I agree, the logic of "show duplicated lines" is of kind confusing to put in a few words... The idea is to keep a unique list of duplicated lines.
e.g.
```
line1
line1
line1
line2
line3
line3
```
"Show duplicated lines":
```
line1
line3
```

So maybe rename as "Sort lines (keep a unique list of duplicated lines)"
and/or create 2 separated commands:
"Sort lines (keep duplicated lines)" and "Sort lines (return unique lines)"

**Review by Tyriar — COMMENTED** [review]
I agree the wording is a bit awkward. After thinking about it, since we already have "remove duplicate lines", I think "keep only duplicate lines" to keep similar wording is the most clear.

**Comment by Tyriar** [comment]
Published v1.10.0

**Pull request merged**